prefect_docker/worker.py

```diff
@@ -535,6 +553,10 @@ def _create_and_start_container(
 
         container = self._create_container(docker_client, **container_settings)
 
+        created_event = self._emit_container_status_change_event(
+            container, configuration
+        )
```

Comment:
If container creation fails, it looks like no events will be emitted. Would it be worthwhile to emit an event if container creation fails?

Reply:
That's a great point, I'll create an issue to track that, thanks @desertaxle!